client/src/pages/Board.js

```diff
@@ -27,7 +28,32 @@ const useStyles = makeStyles(theme => ({
 
 const Board = () => {
   const classes = useStyles();
-  const [data, setData] = useState(initialData);
+  // const [data, setData] = useState(initialData);
+
+  const [switchBoard, setSwitchBoard] = useState(false);
+
+  //Access the states from Dashboard Provider
+  const {
+    isFetching, setIsFetching,
+    error, setError,
+    boards, setBoards,
+    selectedBoard, setSelectedBoard,
+    columns, setColumns,
+    cards, setCards
+  } = useContext(DashboardContext);
+
+
+
+  if (error!== null || isFetching || boards.length ===0) {
+    return (
+        <BoardBar/>
+    )
+  }
+
+  console.log(boards);
```

Comment:
you may want to remove the console log statements here

Reply:
thanks